src/www/ui/api/documentation/openapi.yaml

```
@@ -1735,6 +1822,34 @@ components:
             - INFO
             - ERROR
           description: Denotes if info was created on error
+    Token:
+      type: object
+      properties:
+        id:
+          type: string
+          description: Token Id
+          example: 2.3
+        name:
+          type: string
+          description: Name of the Token
+        created:
+          type: string
```

Please add date format
```suggestion
          type: string
          format: date
```

---

sure!